samples/CallingStateful/src/App.tsx

```diff
@@ -202,26 +207,48 @@ function App(): JSX.Element {
 
   return (
     <FluentThemeProvider>
-      <Stack
-        verticalAlign="center"
-        horizontalAlign="center"
-        tokens={{ childrenGap: '1rem' }}
-        style={{ width: '100%', height: '40rem', margin: 'auto', paddingTop: '1rem', position: 'relative' }}
-      >
-        {userIdentifier && <Text>your userId: {userIdentifier.communicationUserId}</Text>}
-        {teamsIdentifier && <Text>your teamsId: {teamsIdentifier}</Text>}
-        {
-          /* @conditional-compile-remove(one-to-n-calling) */ statefulCallClient && callAgent && !call && (
+      <Stack horizontal tokens={{ childrenGap: '1rem' }} style={{ margin: '1rem', position: 'relative' }}>
+        <Stack style={{ width: '100%', height: '100%' }}>
+          {userIdentifier && <Text>your userId: {userIdentifier.communicationUserId}</Text>}
+          {teamsIdentifier && <Text>your teamsId: {teamsIdentifier}</Text>}
+          {statefulCallClient && callAgent && !call && (
             <HomeScreen callAgent={callAgent as CallAgent} headerImageProps={imageProps}></HomeScreen>
-          )
-        }
-        {statefulCallClient && /* @conditional-compile-remove(one-to-n-calling) */ callAgent && call && (
-          <CallScreen
-            statefulCallClient={statefulCallClient}
-            /* @conditional-compile-remove(one-to-n-calling) */ callAgent={callAgent}
-            /* @conditional-compile-remove(one-to-n-calling) */ call={call}
-            /* @conditional-compile-remove(one-to-n-calling) */ onSetCall={setCall}
-          />
+          )}
+          {statefulCallClient && /* @conditional-compile-remove(one-to-n-calling) */ callAgent && call && (
+            <Stack style={{ height: '40rem', width: '100%' }}>
+              <CallScreen
+                statefulCallClient={statefulCallClient}
+                /* @conditional-compile-remove(one-to-n-calling) */ callAgent={callAgent}
+                /* @conditional-compile-remove(one-to-n-calling) */ call={call}
+              />
+            </Stack>
+          )}
+        </Stack>
+        {calls.length > 0 && (
+          <Stack style={{ width: '20%', height: '100%' }}>
```

Comment:
Be careful setting width to a `%`, these often perform poorly when the screen size is shrunk. Either give a min-width or just set a fixed width